code/modules/mob/living/carbon/human/species/station/prometheans.dm

```diff
@@ -149,10 +150,14 @@ var/datum/species/shapeshifter/promethean/prometheans
 			H.gib()
 
 /datum/species/shapeshifter/promethean/handle_environment_special(var/mob/living/carbon/human/H)
-	var/healing = TRUE	// Switches to FALSE if the environment's bad
-
-	if(H.fire_stacks < 0)	// If you're soaked, you're melting
-		H.adjustToxLoss(2*heal_rate)	// Doubled because 0.5 is miniscule, and fire_stacks are capped in both directions
+	var/healing = TRUE	// Switches to FALSE if healing is not possible at all.
+	var/regen_brute = TRUE
```

Comment:
Why do you insist on keeping these? Keep things simple.

Reply:
All four are present for any present (Low pressure causing physical injuries to stop regenerating) or possible future need to toggle a single type of healing, based on a new chemical effect or modifier or what-have-you, without disabling them all.
It's also there to keep the lower checks ordered so it's easier for me to transfer them into proper organ processes later.